test/src/e2e.long/mempool.test.ts

```diff
@@ -80,6 +81,73 @@ describe("Memory pool size test", function() {
             expect(pendingTransactions.length).to.equal(sizeLimit);
         }).timeout(20_000);
 
+        it("Rejected by limit and reaccepted", async function() {
+            const sent = [];
+            for (let i = 0; i < sizeLimit * 2; i++) {
+                sent.push(
+                    await nodeA.sendPayTx({
+                        seq: i,
+                        awaitInvoice: false
+                    })
+                );
+            }
+            let counter = 0;
+            while (
+                (await nodeB.sdk.rpc.chain.getPendingTransactions()).length <
+                sizeLimit
+            ) {
+                await wait(500);
+                counter += 1;
+            }
+            await wait(500 * (counter + 1));
+
+            let pendingTransactions = await nodeB.sdk.rpc.chain.getPendingTransactions();
+            let pendingTransactionHashes = pendingTransactions.map(
+                (tx: SignedTransaction) => tx.hash().value
+            );
+            const rejectedTransactions = sent.filter(
+                tx => !pendingTransactionHashes.includes(tx.hash().value)
+            );
+
+            await nodeB.sdk.rpc.devel.startSealing();
+
+            counter = 0;
+            while (
+                (await nodeB.sdk.rpc.chain.getPendingTransactions()).length > 0
+            ) {
+                await wait(500);
+                counter += 1;
+            }
+            await wait(500 * (counter + 1));
+
+            await nodeB.sdk.rpc.devel.stopSealing();
+
+            rejectedTransactions.forEach(async (tx: SignedTransaction) => {
```

Comment:
```ts
Promise.all(rejectedTransactions.map((tx: SignedTransaction) => nodeB.sdk.rpc.chain.sendSignedTransaction(tx));
```

Reply:
`forEach` doesn't guarantee the promises are resolved.

Reply:
Fixed